src/main/java/com/iota/iri/conf/Configuration.java

```diff
@@ -92,6 +93,7 @@
         conf.put(DefaultConfSettings.MAIN_DB.name(), "rocksdb");
         conf.put(DefaultConfSettings.EXPORT.name(), "false");
         conf.put(DefaultConfSettings.SEND_LIMIT.name(), "-1.0");
+        conf.put(DefaultConfSettings.NEW_TX_LIMIT.name(), "9.0");
```

Comment:
What is new tx limit?

Reply:
this is a limit on the incoming *new* transactions from 1 neighbor,
it's added to artificially simulate network saturation a case where a node is flooded with transactions.

this limit will be set to "off" by default.